dev/com.ibm.ws.componenttest/src/componenttest/app/AssertionErrorSerializer.java

```diff
@@ -0,0 +1,216 @@
+/*******************************************************************************
+ * Copyright (c) 2021 IBM Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ *     IBM Corporation - initial API and implementation
+ *******************************************************************************/
+package componenttest.app;
+
+import java.io.StringReader;
+import java.io.StringWriter;
+import java.util.ArrayList;
+
+import javax.json.Json;
+import javax.json.JsonArray;
+import javax.json.JsonArrayBuilder;
+import javax.json.JsonObject;
+import javax.json.JsonObjectBuilder;
+import javax.json.JsonReader;
+import javax.json.JsonWriter;
+
+/**
+ * Serializes and Deserializes an AssertionError to JSON
+ */
+public class AssertionErrorSerializer {
```

Comment:
From some of what we discussed we cannot use the transformer since the transformed jar is not available to be added to classpaths.  I think we should try to use transformer in the future if possible though once we get some of the technical issues resolved.